### PR TITLE
Revise web JS linter rules

### PR DESCRIPTION
**Pull request by benbuckman**
Prompted by unnecessary linter complaints in https://github.com/Shyp/shyp.com/pull/157/files.

### CODE REVIEW

**Comment by dylanpyle** [comment]
LGTM!

File: index.js [hunk 1]
```diff
@@ -106,7 +106,7 @@ const stylistic = {
   'eol-last': 2,
 
   // Two-space indentation, always
-  indent: [2, 2],
+  indent: [2, 2, {'SwitchCase': 1}],
```
Comment (dylanpyle) [inline index.js]: Technically the key doesn't need to be quoted here - if were were running this ruleset against itself, it would complain

Comment (benbuckman) [inline index.js]: Good point. Will change on master.

Comment (benbuckman) [inline index.js]: 289843e

**Pull request merged**